conf/scalardb-cluster-custom-values-direct-kubernetes-mode.yaml

```diff
@@ -56,33 +56,21 @@ scalardbCluster:
       cpu: 2000m
       memory: 4Gi
 
-  tolerations:
-    - effect: NoSchedule
-      key: scalar-labs.com/dedicated-node
-      operator: Equal
-      value: scalardb-cluster
```

Comment:
This `tolerations` configuration was used for the old too strict recommendations (`1 ScalarDB Cluster pod on 1 worker node`). Now, we removed these `too strict recommendations`, so we can remove this configuration from this sample config file.